parent/pom.xml

```diff
@@ -40,6 +40,12 @@
       <artifactId>spotbugs</artifactId>
       <version>3.1.12</version>
     </dependency>
+
+    <dependency>
+      <groupId>org.ejml</groupId>
+      <artifactId>ejml-all</artifactId>
```

Comment:
is this change needed? I think adding the dependency to the ripc pom file should be sufficient